src/frontend/.storybook/preview.tsx

```diff
@@ -34,7 +35,9 @@ export const decorators = [
   (Story) => (
     <IntlProvider locale="en">
       <IconsWrapper>
-        <Story />
+        <CunninghamProvider>
+          <Story />
+        </CunninghamProvider>
```

Comment:
Did you check that some stories does not import CunninghamProvider internally ?

Reply:
i hadn't, it's done now some stories needed to have the provider removed.